internal/exec/engine.go

```diff
@@ -24,6 +24,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/google/renameio"
```

Comment:
Very minor: should probably move this import down to go with the two other non-stdlib external imports (the `vcontext` imports)

Reply:
Done